libnmstate/schemas/operational-state.yaml

```diff
@@ -287,6 +287,23 @@ definitions:
                     type: string
                   access-tag:
                     type: string
+                link-aggregation:
+                  type: object
+                  required:
+                  - mode
+                  - slaves
```

Comment:
If both are required, we cannot do partial editation (now or in the future).
I suggest you leave it up to the implementation to validate this, per its ability.

Reply:
Ack.